src/ucp/core/ucp_worker.c

```diff
@@ -2059,11 +2047,14 @@ void ucp_worker_create_vfs(ucp_context_h context, ucp_worker_h worker)
 {
     ucs_thread_mode_t thread_mode;
 
+    UCS_ASYNC_BLOCK(&worker->async);
```

Comment:
no point to tack worker lock when adding entry to VFS
it should be like this
```
ucp_worker_vfs_show_primitive(...)
{
    worker = obj;
    UCS_ASYNC_BLOCK(&worker->async);
    ucs_vfs_show_primitive(obj, arg_ptr, arg_u64, ...)
    UCS_ASYNC_UNBLOCK(&worker->async);
}
...
{
    ... (no lock)
    ucs_vfs_obj_add_ro_file(worker, ucp_worker_vfs_show_primitive,
             &worker->keepalive.round_count, UCS_VFS_TYPE_SIZET,
             "keepalive/round_count");
}

```

Reply:
done